### PR TITLE
[BAHIR-285] flink-connector-redis support SREM command.

### PR DESCRIPTION
**Pull request by wyxpku**
Support redis SREM command.

### CODE REVIEW

File: .github/workflows/maven-ci.yml [hunk 1]
```diff
@@ -45,4 +45,4 @@ jobs:
       run: ./dev/change-scala-version.sh ${{ matrix.scala-version }}
       shell: bash
     - name: Build with flink ${{ matrix.flink-version }}
-      run: mvn -B clean verify -Dscala-${{ matrix.scala-version }} -Dflink.version=${{ matrix.flink-version }}
```
Comment (eskabetxe) [inline .github/workflows/maven-ci.yml:48]: why?

File: flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java [hunk 2]
```diff
@@ -77,14 +77,20 @@ public void testRedisListDataType() throws Exception {
...
             new RedisCommandMapper(RedisCommand.SADD));
 
-        source.addSink(redisSink);
-        env.execute("Test Redis Set Data Type");
+        source.addSink(redisSaddSink);
+        env.execute("Test SADD");
 
         assertEquals(NUM_ELEMENTS, jedis.scard(REDIS_KEY));
 
+        RedisSink<Tuple2<String, String>> redisSremSink = new RedisSink<>(jedisPoolConfig,
```
Comment (eskabetxe) [inline flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java:88]: could we add a new test only to remove?

**Review by eskabetxe — CHANGES_REQUESTED** [review]
Could you separate in two commits one for each command added?

File: flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java (hunk 2 above)
Comment (wyxpku) [reply flink-connector-redis/src/test/java/org/apache/flink/streaming/connectors/redis/RedisSinkITCase.java:88]: > Could you separate in two commits one for each command added?

Sorry, It's a mistake, I'm new to github. 😅
I will close this PR and create a new one in future.

Thanks for review.

**Pull request closed without merging**